projects/kit/components/input-tag/input-tag.template.html

```diff
@@ -100,6 +100,7 @@
                                 [tuiFocusable]="computedFocusable"
                                 [ngModel]="search"
                                 (ngModelChange)="onInput($event)"
+                                (paste.prevent)="onPaste($event)"
```

Comment:
Technically we could also handle `drop` the same way. Could you give it a spin?

Reply:
done ✅